locksmith/package.json

```diff
@@ -34,7 +28,7 @@
   "dependencies": {
     "@sentry/node": "6.12.0",
     "@sentry/tracing": "6.12.0",
-    "@unlock-protocol/networks": "0.0.2",
+    "@unlock-protocol/networks": "workspace:./packages/networks",
```

Comment:
Ohhh! I was wondering what was the way to do that :)

Reply:
yes thats yarn v2